packages/nimble-components/src/menu-button/specs/README.md

```diff
@@ -84,6 +84,9 @@ _Methods_
 
 _Events_
 
+-   `beforetoggle` (event) - event fired before the opened state has changed. The event detail contains:
+    -   `newState` - boolean - The value of `open` on the menu button that the element is transitioning in to.
+    -   `oldState` - boolean - The value of `open` on the menu button that the element is transitioning out of.
 -   `open-change` (event) - event for when the opened state has changed
```

Comment:
From [spec comments](https://github.com/ni/nimble/pull/956#discussion_r1073042699) I think we aligned on having `beforetoggle` and `toggle` events instead of `open-change`. Do we plan to do that in the future? Should we remove open-change now or only when switching to toggle?

Reply:
I was hesitant to make that change since it would be a breaking change. I was pretty sure we were using that event in SLE, but it turns out we aren't. I'll go ahead and rename it as part of this PR.